staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -52,7 +51,7 @@ const (
 	triesBeforeBackOff = 1
 )
 
-var createPatchErrFormat = "creating patch with:\noriginal:\n%s\nmodified:\n%s\ncurrent:\n%s\nfor:"
+var createPatchErrFormat = "creating patch with:\noriginal:\n%s\nmodified:\n%s\ncurrent:\n%s\nfor:, err: %w"
```

Comment:
`for:, err:` seems unexpected. Are we missing a format directive after `for:`?

Reply:
Apologies, made a mistake while adding this. I'll correct it.